src/components/LayerSwitcher/SuccessLayerInput.tsx

```diff
@@ -0,0 +1,103 @@
+import { Autocomplete } from '@material-ui/lab';
+import {
+  Box,
+  Checkbox,
+  FormControlLabel,
+  FormGroup,
+  TextField,
+} from '@material-ui/core';
+import * as React from 'react';
+import { Star } from '@material-ui/icons';
+import { LayerIndex } from './helpers/loadLayers';
+import { isViewInsideBbox } from './helpers';
+import { useMapStateContext } from '../utils/MapStateContext';
+
+type SuccessLayerDataInputProps = {
+  index: LayerIndex[];
+  onSelect: (layer: LayerIndex | undefined) => void;
+};
+
+const getLayerLabel = ({ name, country_code }: LayerIndex) => {
+  if (country_code) return `${country_code} - ${name}`;
+
+  return `Global - ${name}`;
+};
+
+export const SuccessLayerInput: React.FC<SuccessLayerDataInputProps> = ({
+  index,
+  onSelect,
+}) => {
+  const [showAllowedForOsm, setShowAllowedForOsm] = React.useState(true);
+  const [currentlyVisible, setCurrentlyVisible] = React.useState(true);
+  const { view } = useMapStateContext();
+
+  return (
+    <div>
+      <FormGroup>
+        <FormControlLabel
+          control={
+            <Checkbox
+              checked={showAllowedForOsm}
+              onChange={({ target: { checked } }) =>
+                setShowAllowedForOsm(checked)
+              }
+            />
+          }
+          label="Allowed for the use in osm"
+        />
+        <FormControlLabel
+          control={
+            <Checkbox
+              checked={currentlyVisible}
+              onChange={({ target: { checked } }) =>
+                setCurrentlyVisible(checked)
+              }
+            />
+          }
+          label="Only currently visible layers"
```

Comment:
Could we change the text to "Filter layers in this area"? and add it to vocabulary pls

Also can you please move the checkbox below the Autocomplete and make it smaller? It is a little confusing showing it so prominently. 
Layout perhaps sth like this:
![image](https://github.com/zbycz/osmapp/assets/385047/318c35da-b58a-4718-9a69-49cd3c7e1072)

Reply:
![image](https://github.com/user-attachments/assets/3e5d2eb9-659c-44e9-b43e-324a8047d1e1)
Like this?

Reply:
yes, that is perfect! 👍